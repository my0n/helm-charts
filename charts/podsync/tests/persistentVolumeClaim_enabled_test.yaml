suite: test persistentVolumeClaim
templates:
  - persistentVolumeClaim.yaml
tests:
  - it: should match snapshot
    set:
      persistence.enabled: true
    asserts:
      - matchSnapshot: {}
