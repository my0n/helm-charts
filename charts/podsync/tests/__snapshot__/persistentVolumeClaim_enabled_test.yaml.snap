should match snapshot:
  1: |
    apiVersion: v1
    kind: PersistentVolumeClaim
    metadata:
      annotations: {}
      name: RELEASE-NAME-podsync
    spec:
      accessModes:
      - ReadWriteOnce
      resources:
        requests:
          storage: 10Gi
      storageClassName: null
      volumeMode: Filesystem
